src/core/ext/filters/client_channel/resolver/xds/xds_resolver.cc

```diff
@@ -419,6 +421,9 @@ bool UnderFraction(const uint32_t fraction_per_million) {
 
 ConfigSelector::CallConfig XdsResolver::XdsConfigSelector::GetCallConfig(
     GetCallConfigArgs args) {
+  std::string blah = "blah";
```

Comment:
Please remove.

Reply:
fixed

Reply:
removed real code to use xxhash is coming as soon as this is submitted